js/ftx.js

```diff
@@ -2409,7 +2409,7 @@ module.exports = class ftx extends Exchange {
         const initialMargin = Precise.stringMul (notionalString, initialMarginPercentage);
         const maintenanceMarginPercentageString = this.safeString (position, 'maintenanceMarginRequirement');
         const maintenanceMarginString = Precise.stringMul (notionalString, maintenanceMarginPercentageString);
-        const unrealizedPnlString = this.safeString (position, 'recentPnl');
+        const unrealizedPnlString = this.safeString (position, 'unrealizedPnl');
```

Comment:
```suggestion
        const unrealizedPnlString = this.safeString2 (position, 'unrealizedPnl', 'recentPnl');
```

Reply:
@ttodua it has to be tested:

<img width="1675" alt="Screenshot 2022-07-07 at 23 35 24" src="https://user-images.githubusercontent.com/1294454/177866767-adf63868-52c4-471a-94fc-e58c4e602196.png">

Reply:
I've seen that field in api docs, but i meant, there is no explanation of that field. when I've tested that today earlier, it returned null for my position/s. i'll dig into it further and let you know